presto-accumulo/src/main/java/com/facebook/presto/accumulo/AccumuloMetadata.java

```diff
@@ -142,9 +142,17 @@ public void renameTable(ConnectorSession session, ConnectorTableHandle tableHand
         client.renameTable(handle.toSchemaTableName(), newTableName);
     }
 
+    @Deprecated
```

Comment:
You should simply remove this method. It has a default implementation in the interface that throws an `UnsupportedOperationException`.  Same for other connectors.